mk/config.mk

```diff
@@ -327,6 +327,13 @@ endif
 # Enable paging, requires SRAM, can't be enabled by default
 CFG_WITH_PAGER ?= n
 
+# Use the pager for user TAs
+CFG_PAGED_USER_TA ?= $(CFG_WITH_PAGER)
+
+# If paging of user TAs, that is, R/W paging default to eneble paging of
+# TAG and IV in order to reduce heap usage.
+CFG_CORE_PAGE_TAG_AND_IV ?= ${CFG_PAGED_USER_TA}
```

Comment:
`()` not `{}`

Reply:
Thanks, I usually make that error the other way around in shell scripts. :-)